azure/README.md

```diff
@@ -1,17 +1,17 @@
 # Setup instructions
 
-- Install packer by following the instructions in the following [link](https://learn.hashicorp.com/tutorials/packer/get-started-install-cli).
+- Install packer by the [following](https://learn.hashicorp.com/tutorials/packer/get-started-install-cli) instructions.
```

Comment:
Thanks for changing this line :-). 

Can you please add a minor change to the sentence as shown below:
Install packer by *using* the following instructions.